src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,22 @@
+package racingcar;
+
+public class Car {
+
+    private int position = 0;
+
+    public void move(int distance) {
+        if (judgeForth(distance)) position++;
```

Comment:
```suggestion
        if (judgeForth(distance)) {
            position++;
        }
```
한줄 if문이 아닌 이런식으로 작성해주는게 더 보기 좋은거 같아요. (개인적인 의견)